packages/composite-checkout/src/components/checkout-step.js

```diff
@@ -138,26 +147,21 @@ function CheckIcon() {
 				/>
 			</mask>
 			<g mask="url(#mask1)">
-				<rect width="20" height="20" fill="#008A20" />
+				<rect width="20" height="20" fill="white" />
 			</g>
 		</svg>
 	);
 }
 
 const StepWrapper = styled.div`
 	padding-bottom: ${props => ( props.finalStep ? '0' : '32px' )};
-	margin-bottom: 8px;
 	position: relative;
+	border-bottom: ${props => ( props.finalStep ? '0' : '1px' )} solid
```

Comment:
Does having two functions here work? Maybe it does. I'm unclear on some of the details about this pattern in styled-components. Anyway, if there's an issue, I think this could also be:

```
border-bottom: ${({finalStep, theme}) => (finalStep ? '0' : '1px solid ' + theme.colors.borderColorLight)}
```

(Particularly since a 0 width border doesn't need a color.)